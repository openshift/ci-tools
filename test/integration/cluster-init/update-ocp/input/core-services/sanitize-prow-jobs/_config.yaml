buildFarm:
  aws:
    build01:
      filenames:
        - some-yaml-file.yaml
    build02:
      filenames:
        - some-other-yaml-file.yaml
default: build01
groups:
  app.ci:
    jobs:
      - some-job
      - pull-ci-openshift-release-master-existingCluster-dry
  build01:
    jobs:
      - some-other-job
kvm:
  - build02
sshBastion: build01