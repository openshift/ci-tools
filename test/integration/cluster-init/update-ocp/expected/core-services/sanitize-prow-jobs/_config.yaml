buildFarm:
  aws:
    build01:
      filenames:
      - some-yaml-file.yaml
    build02:
      filenames:
      - some-other-yaml-file.yaml
default: build01
groups:
  app.ci:
    jobs:
    - branch-ci-openshift-release-master-existingCluster-apply
    - periodic-openshift-release-master-existingCluster-apply
    - pull-ci-openshift-release-master-existingCluster-dry
    - some-job
  build01:
    jobs:
    - some-other-job
kvm:
- build02
sshBastion: build01
